common/lcs/src/error.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
@bmwill I suspect you could reduce this file by 80% by using https://github.com/dtolnay/thiserror

Reply:
True, it may be better to use thiserror, but I think that the file size may still end up being roughly the same length :P